server/config/config.go

```diff
@@ -1,5 +1,41 @@
 package config
 
```

Comment:
Now that I think about it, what is the primary motivation here for having a new config? So `.gaiad/config` will have `config.toml` and `cosmos.toml`. Do we see this `cosmos.toml` config growing overtime? Or can `minimum_fees` be added to TM's `config.toml`?

Not a big deal either way, just thinking about how other users will think and feel.

Reply:
Fair enough! 👍